### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by squelix**
Add libx11 to get karma work

### CODE REVIEW

**Comment by smaftoul** [comment]
Please use a correct commit message (you can when you edit in github) ! ;)

File: Dockerfile [hunk 1]
```diff
@@ -5,7 +5,9 @@ RUN /usr/local/bin/install-plugins.sh < /usr/share/jenkins/ref/plugins.txt
 USER root
 RUN groupadd -g 999 docker \
     && adduser jenkins docker \
-    && apt-get update && apt-get install -y libltdl7 \
+    && apt-get update \
+    && apt-get install -y libltdl7 \
+    && apt-get install -y libx11-xcb1 \
```
Comment (smaftoul) [inline Dockerfile]: apt-get update && apt-get install -y libltdl7 libx11-xcb1 ?

**Review by smaftoul — COMMENTED** [review]
Anyway, what is karma ? can this be containrized ?

**Comment by squelix** [comment]
On a besoin de karma pour lancer les tests sur les projets angular, donc on a un chromeHeadless qui a besoin de libx11, d'ou le apt-get.

**Comment by squelix** [comment]
Good ?

File: Dockerfile [hunk 2]
```diff
@@ -5,7 +5,8 @@ RUN /usr/local/bin/install-plugins.sh < /usr/share/jenkins/ref/plugins.txt
 USER root
 RUN groupadd -g 999 docker \
     && adduser jenkins docker \
-    && apt-get update && apt-get install -y libltdl7 \
+    && apt-get update \
```
Comment (remialvado) [inline Dockerfile:8]: y'a vraiment besoin de 2 `apt-get update` ? :)

Comment (remialvado) [reply Dockerfile:8]: A part ce double `apt-get update`, le reste me parait bon.

**Comment by smaftoul** [comment]
Nope, not good, karma is an external tool , it doesn't belong to our CI , we must use a containerized step in our Jenkinsfile !

**Pull request closed without merging**